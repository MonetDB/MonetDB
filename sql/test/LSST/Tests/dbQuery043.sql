-- http://dev.lsstcorp.org/trac/wiki/dbQuery043 
SELECT  S1.movingObjectId AS s1, 
        S2.movingObjectId AS s2
FROM    MovingObject S1,                                   -- S1 is the white dwarf
        MovingObject S2,                                   -- S2 is the second star
		DiaSource E1, DiaSource E2
WHERE   E1.extendedness < 0.2                       -- is star
   AND  E2.extendedness < 0.2                       -- is star
  AND E1.diaSourceId = S1.movingObjectId
  AND E2.diaSourceId = S2.movingObjectId
--   AND  spDist(S1.ra, S1.decl, S2.ra, S2.decl) < .05 -- the 5 arcsecond test
   AND  S1.uMag-S1.gMag < 0.4                        -- and S1 meets Paul Szkody's color cut
   AND  S1.gMag-S1.rMag < 0.7                        -- for white dwarfs
   AND  S1.rMag-S1.iMag > 0.4 
   AND  S1.iMag-S1.zMag > 0.4;
