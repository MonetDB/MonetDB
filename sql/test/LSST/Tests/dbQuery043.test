query II rowsort
SELECT  S1.movingObjectId AS s1,
        S2.movingObjectId AS s2
FROM    MovingObject S1,
        MovingObject S2,
		DiaSource E1, DiaSource E2
WHERE   E1.extendedness < 0.2
   AND  E2.extendedness < 0.2
  AND E1.diaSourceId = S1.movingObjectId
  AND E2.diaSourceId = S2.movingObjectId
   AND  S1.uMag-S1.gMag < 0.4
   AND  S1.gMag-S1.rMag < 0.7
   AND  S1.rMag-S1.iMag > 0.4
   AND  S1.iMag-S1.zMag > 0.4
----

