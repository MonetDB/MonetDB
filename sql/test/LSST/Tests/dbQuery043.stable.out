stdout of test 'dbQuery043` in directory 'sql/test/LSST` itself:


# 05:56:06 >  
# 05:56:06 >   mserver5 "--config=/ufs/mk/current//Linux/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/ufs/mk/current//Linux/lib/MonetDB5:/ufs/mk/current//Linux/lib/MonetDB5/lib:/ufs/mk/current//Linux/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/mk/current//Linux/var/MonetDB5/dbfarm"  --set mapi_open=true --set xrpc_open=true --set mapi_port=34860 --set xrpc_port=47662 --set monet_prompt= --trace --forcemito --set mal_listing=2  "--dbname=mTests_src_test_LSST" --set mal_listing=0 ; echo ; echo Over..
# 05:56:06 >  

# MonetDB server v5.23.0, based on kernel v1.41.0
# Not released
# Serving database 'mTests_src_test_LSST', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 7.750 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://eir.ins.cwi.nl:34860/
# MonetDB/SQL module v2.41.0 loaded
# MonetDB/GIS module v0.21.0 loaded



# 05:56:06 >  
# 05:56:06 >  mclient -lsql -ftest -i -e --host=eir --port=34860 
# 05:56:06 >  

#SELECT  S1.movingObjectId AS s1, 
#        S2.movingObjectId AS s2
#FROM    MovingObject S1,                                   -- S1 is the white dwarf
#        MovingObject S2,                                   -- S2 is the second star
#		DiaSource E1, DiaSource E2
#WHERE   E1.extendedness < 0.2                       -- is star
#   AND  E2.extendedness < 0.2                       -- is star
#  AND E1.diaSourceId = S1.movingObjectId
#  AND E2.diaSourceId = S2.movingObjectId
#--   AND  spDist(S1.ra, S1.decl, S2.ra, S2.decl) < .05 -- the 5 arcsecond test
% sys.,	sys. # table_name
% s1,	s2 # name
% bigint,	bigint # type
% 1,	1 # length

# 05:56:06 >  
# 05:56:06 >  Done.
# 05:56:06 >  

